streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -701,18 +703,33 @@ private KafkaStreams(final InternalTopologyBuilder internalTopologyBuilder,
                 internalTopologyBuilder,
                 parseHostInfo(config.getString(StreamsConfig.APPLICATION_SERVER_CONFIG)));
 
+        final int numStreamThreads;
+        if (internalTopologyBuilder.hasNoNonGlobalTopology()) {
+            log.warn("Overriding number of StreamThreads to zero for global-only topology");
```

Comment:
I don't think this should be WARN but an INFO. There is nothing "wrong" and thus nothing to warn about?